src/std-traits/closures.md

```diff
@@ -52,14 +54,16 @@ When you define a function that takes a closure, you should take `FnOnce` if you
 can (i.e. you call it once), or `FnMut` else, and last `Fn`. This allows the
 most flexibility for the caller.
 
-In contrast, when you have a closure, the most flexible you can have is `Fn` (it
-can be passed everywhere), then `FnMut`, and lastly `FnOnce`.
+In contrast, when you have a closure, the most flexible you can have is `Fn`
+(which can be passed to a consumer of any of the 3 closure traits), then
+`FnMut`, and lastly `FnOnce`.
 
 The compiler also infers `Copy` (e.g. for `add_3`) and `Clone` (e.g.
-`multiply_sum`), depending on what the closure captures.
+`multiply_sum`), depending on what the closure captures. Function pointers
+(references to `fn` items) implement `Copy` and `Fn`.
 
-By default, closures will capture by reference if they can. The `move` keyword
-makes them capture by value.
+By default, closures will capture each variable from an outer scope by the least
+demanding form of access they can. The `move` keyword forces capture by value.
```

Comment:
"demanding" might be a confusing term here -- perhaps a parenthetical to enumerate the options would help, especially in translations